cob_sound/action/Play.action

```diff
@@ -0,0 +1,6 @@
+# goal
+std_msgs/String filename
+---
+#result
```

Comment:
If we can evaluate in the ActionServer, whether the command has successfully been executed / or failed due to a certain failure...I guess it would be nice to send this in the result...just because this is the purpose of the result part of the message...